spec/lib/constants_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+RSpec.describe Constants do
```

Comment:
Not sure... the intent here is that if someone updated the constant they would also need to update the spec. Making sure that someone didn't remove one and know that it was actually expected to be there.